source/simulator/assemblers/advection.cc

```diff
@@ -437,11 +446,6 @@ namespace aspect
                 }
             }
         }
-      else if (cell->has_periodic_neighbor (face_no))
```

Comment:
why remove this?

Reply:
This logic is now in assembly.cc:1040 (i.e. before we call the assembler for a face we check if there is a periodic neighbor, and then immediately call the other assembler).